src/plugins/data/public/ui/filter_bar/_global_filter_group.scss

```diff
@@ -55,3 +55,12 @@
     margin-top: $euiSize * -1;
   }
 }
+
+.globalFilterGroup__removeAllFilters {
```

Comment:
not sure what element this class is on, but does that element have a component prop we can set to get danger styling?

Reply:
It's used in the one of the filter options "remove all"
![image](https://github.com/user-attachments/assets/5c396d29-8984-43f9-974e-b959f6b5afd4)

The element prop is `EuiContextMenuItemProps`, which doesn't comes with a styling prop.
![image](https://github.com/user-attachments/assets/f50393af-ce11-4c24-92db-a3ea07f3f263)